app/i18n/tr/feedback.php

```diff
@@ -1,27 +1,27 @@
 <?php
 
 /******************************************************************************/
-/* Each entry of that file can be associated with a comment to indicate its   */
-/* state. When there is no comment, it means the entry is fully translated.   */
-/* The recognized comments are (comment matching is case-insensitive):        */
-/*   + TODO: the entry has never been translated.                             */
-/*   + DIRTY: the entry has been translated but needs to be updated.          */
-/*   + IGNORE: the entry does not need to be translated.                      */
-/* When a comment is not recognized, it is discarded.                         */
+/* Bu dosyanın her bir girdisi, durumunu belirtmek için bir yorumla ilişkilendirilebilir. */
+/* Yorum yoksa, bu, girdinin tamamen çevrildiğini gösterir.                    */
```

Comment:
A bit late, but the comments don't need to be translated (and it's one of the reasons the automated test is unhappy).